Original call: cluster_error -d /home/alex/Dropbox/keith/projects/mbgdml-solvents/data/datasets/meoh/1meoh/62meoh.sphere.gfn2.md.500k.prod1.3meoh.cm14.dset.1meoh-dset.npz -i /home/alex/Dropbox/keith/projects/mbgdml-solvents/data/models/meoh/1meoh/62meoh.sphere.gfn2.md.500k.prod1.3meoh.cm14.dset.1meoh-model-iterativetrain1000.npz -c /home/alex/Dropbox/keith/projects/mbgdml-solvents/analysis/iterative-training/meoh/1meoh/62meoh.sphere.gfn2.md.500k.prod1.3meoh.cm14.dset.1meoh-model-randomtrain200/saves/cluster_error_62meoh/cluster_indices.npy
